### PR TITLE
[Delivers #187355094]A successfully registered user should receive an…

### PR DESCRIPTION
**Pull request by venuste-20002**
#### What does this PR do?
 A successfully registered user should receive an email to verify their account email
#### Description of Task to be completed?
by verifying you account you will provide you registering format role and you will receive the email said that you are successful register with the link to verify. 
#### How should this be manually tested?
after clone and install all the package 
- run using npm dev 
- using postman (localhost:7000/api/users/register)

#### What are the relevant pivotal tracker/Trello stories?
[pivotal tracker link ](https://www.pivotaltracker.com/story/show/187355094)

### CODE REVIEW

**Review by lilika67 — APPROVED** [review]

File: coverage/clover.xml
Comment (UmuhireJessie) [inline coverage/clover.xml:1]: Remove coverage folder

File: src/database/seeders/20240415080213-demo-user.ts [hunk 1]
```diff
@@ -10,13 +10,8 @@ module.exports = {
...
           lastName: 'Ishimwe',
-          email: 'christianinja3@gmail.com',
-        },
-        {
-          userId: uuid(),
-          firstName: 'celestin',
-          lastName: 'Nshuti',
-          email: 'nshuticelestin@gmail.com',
+          email: 'christelnjeiauininja3@gmail.com',
+          password: '',
```
Comment (UmuhireJessie) [inline src/database/seeders/20240415080213-demo-user.ts]: Whatever password you give to the seed user should be hashed before getting into the database

File: src/routes/exampleRoutes.ts
Comment (UmuhireJessie) [inline src/routes/exampleRoutes.ts:1]: Avoid committing unused/unimportant code. Remove them instead if you are not using them

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Good work @venuste-20002, consider:
1. working on the comments given
2. Response is not correct when I send bad request, here is the screenshot. 
Screenshot 2024-05-07 at 13 31 20

**Review by Celestin4 — APPROVED** [review]

File: coverage/clover.xml
Comment (UmuhireJessie) [inline coverage/clover.xml:1]: Remove coverage folder

Comment (UmuhireJessie) [inline coverage/clover.xml:1]: Remove coverage folder

Comment (UmuhireJessie) [inline coverage/clover.xml:1]: Remove coverage folder

**Review by igorihimbazwe — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**